player/lua/defaults.lua

```diff
@@ -589,4 +589,14 @@ function mp_utils.getcwd()
     return mp.get_property("working-directory")
 end
 
+function mp_utils.bytes_to_string(b)
```

Comment:
Good, but I think a better name would be something like `format_bytes_humanize`, since this doesn’t “stringify” a byte array.

Reply:
Ah, tbh I didn't even consider it could be interpreted like that. I went for "humanized" as "human-readable" is a tad long.